### ISSUE TITLE
wrk simple bench got error when run a second round on macOS 

### ISSUE THREAD

**Issue by tempbottle** [denoland/deno#3105]
simple http server app.ts:
```
import { serve } from "./server.ts";

const addr =  "127.0.0.1:9999";
const server = serve(addr);
const body = new TextEncoder().encode("Hello World");

async function main(): Promise<void> {
  console.log(`http://${addr}/`);
  for await (const req of server) {
    req.respond({ body });
  }
}

main();
```
run deno -A app.ts, everything is ok.

wrk cmd:
`wrk -t10 -c100 -d10s http://127.0.0.1:9999/  `
first time everything is ok.
and run wrk cmd again:
`wrk -t10 -c100 -d10s http://127.0.0.1:9999/  `
This time get a `0/0 success` ,  Find out the deno process have very light cpu load. Looks like deno have not received any http request.

OS: macOS Catalina 10.15, latest update
deno: 0.20

**Comment by ry** [comment]
It's probably due to your ephemeral ports getting used up.

Try running this:
```
sudo sysctl -w net.inet.ip.portrange.first=32768
sudo sysctl -w net.inet.ip.portrange.hifirst=32768
sudo sysctl -w net.inet.tcp.msl=1000
```

**Comment by tempbottle** [comment]
do some opt for net.inet as:

```
sudo sysctl -w net.inet.ip.portrange.first=32768    
sudo sysctl -w net.inet.ip.portrange.hifirst=32768    
sudo sysctl -w net.inet.tcp.msl=1000   
``` 

and nothing changed.


run `deno -A app.ts` and `cargo run --release -- 9998` to start apps.

run `wrk -c 1000 -t 100 -d10s http://127.0.0.1:9999/` and `wrk -c 1000 -t 100 -d10s http://127.0.0.1:9998/` many round and deno is hang while hyper_hello is good.

use 
```
lsof -i:9999 | awk '{print $1,$2,$4,$5,$6,$7,$8,$9}'
lsof -i:9998 | awk '{print $1,$2,$4,$5,$6,$7,$8,$9}'
```
to make a port status diff, show something intersting:
```
COMMAND PID FD TYPE DEVICE SIZE/OFF NODE NAME
deno 62276 30u IPv4 0xf1537d5cf6d1de85 0t0 TCP localhost:distinct
deno 62276 31u IPv4 0xf1537d5cf6e6cbc5 0t0 TCP localhost:distinct->localhost:53669
deno 62276 32u IPv4 0xf1537d5ceec662c5 0t0 TCP localhost:distinct->localhost:52623
deno 62276 33u IPv4 0xf1537d5cf6ad62c5 0t0 TCP localhost:distinct->localhost:52624
deno 62276 34u IPv4 0xf1537d5cf7047645 0t0 TCP localhost:distinct->localhost:52625
deno 62276 35u IPv4 0xf1537d5cf6320905 0t0 TCP localhost:distinct->localhost:52626
deno 62276 36u IPv4 0xf1537d5cf7043bc5 0t0 TCP localhost:distinct->localhost:52628
deno 62276 37u IPv4 0xf1537d5cf6b89205 0t0 TCP localhost:distinct->localhost:52631
deno 62276 38u IPv4 0xf1537d5cf6e63845 0t0 TCP localhost:distinct->localhost:52632
deno 62276 39u IPv4 0xf1537d5cf6eed645 0t0 TCP localhost:distinct->localhost:52633
deno 62276 40u IPv4 0xf1537d5cf6eecc85 0t0 TCP localhost:distinct->localhost:52634
deno 62276 41u IPv4 0xf1537d5cf6eee005 0t0 TCP localhost:distinct->localhost:52635
deno 62276 42u IPv4 0xf1537d5cf6ee6b05 0t0 TCP localhost:distinct->localhost:52636
deno 62276 43u IPv4 0xf1537d5cf6eec2c5 0t0 TCP localhost:distinct->localhost:52637
deno 62276 44u IPv4 0xf1537d5cf6f7ae85 0t0 TCP localhost:distinct->localhost:52638
deno 62276 45u IPv4 0xf1537d5cf6f7fc85 0t0 TCP localhost:distinct->localhost:52639
deno 62276 46u IPv4 0xf1537d5cf6f81005 0t0 TCP localhost:distinct->localhost:52640
deno 62276 47u IPv4 0xf1537d5cf6f7df45 0t0 TCP localhost:distinct->localhost:52641
deno 62276 48u IPv4 0xf1537d5ceec64f45 0t0 TCP localhost:distinct->localhost:52642
deno 62276 49u IPv4 0xf1537d5ceec63205 0t0 TCP localhost:distinct->localhost:52643
deno 62276 50u IPv4 0xf1537d5ceec65905 0t0 TCP localhost:distinct->localhost:52644
deno 62276 51u IPv4 0xf1537d5cf6a03845 0t0 TCP localhost:distinct->localhost:52645
deno 62276 52u IPv4 0xf1537d5cf6a05585 0t0 TCP localhost:distinct->localhost:52646
deno 62276 53u IPv4 0xf1537d5cf6a04205 0t0 TCP localhost:distinct->localhost:52647
deno 62276 54u IPv4 0xf1537d5cf6d54f45 0t0 TCP localhost:distinct->localhost:52648
deno 62276 55u IPv4 0xf1537d5cf6d53205 0t0 TCP localhost:distinct->localhost:52649
.......
```

```
COMMAND PID FD TYPE DEVICE SIZE/OFF NODE NAME
hyper_hel 61486 3u IPv4 0xf1537d5cf6f7c205 0t0 TCP localhost:distinct32
```

hyper_hello's code:[tools/hyper_hello](https://github.com/denoland/deno/blob/master/tools/hyper_hello/hyper_hello.rs) 

I think the behavior of deno is abnormal here.

**Comment by ry** [comment]
Potentially fixed in 65e9179 ...

**Comment by tempbottle** [comment]
well done！
test with deno 0.23 and everything is ok.
